src/modules/navigator/mission_block.cpp

```diff
@@ -285,6 +287,9 @@ MissionBlock::mission_item_to_position_setpoint(const struct mission_item_s *ite
 
 	case NAV_CMD_LAND:
 		sp->type = position_setpoint_s::SETPOINT_TYPE_LAND;
+		if(_navigator->get_vstatus()->is_vtol){
+			sp->disable_mc_yaw_control = true;
```

Comment:
It will look funny if the system is constantly yawing on land. It should instead be "soft yaw control" or something like that.